package.json

```diff
@@ -35,8 +35,8 @@
     "example": "cd example && ./../node_modules/.bin/babel-node app.js",
     "coveralls": "cat ./coverage/lcov.info | coveralls",
     "prepublish": "npm run lint && npm run build",
-    "lint": "./node_modules/.bin/eslint ./src",
-    "test": "./node_modules/.bin/babel-node -- ./node_modules/.bin/babel-istanbul cover --report lcov _mocha "
+    "lint": "eslint ./src",
+    "test": "babel-node -- ./node_modules/.bin/babel-istanbul cover --report lcov _mocha "
```

Comment:
Ah yes, so it does. Once everything is sorted es6 wise we can review the tests/coverage etc.